shaders/CompoundCloudPlane.gdshader

```diff
@@ -13,9 +13,9 @@ uniform vec4 colour4 : source_color = vec4(0, 0, 0, 0);
 uniform vec2 UVOffset = vec2(0, 0);
 
 // This must evenly divide into 1 or tiling will break
-uniform float NoiseScale = 0.05;
+uniform float NoiseScale = 14;
 
-uniform float BrightnessMultiplier = 1.0;
+uniform float BrightnessMultiplier = 1.2;
```

Comment:
Has the new brightness multiplier been tested in all patches? I ask because many patches have patch specific brightness multipliers defined in `biomes.json`